nj-derive/ui-tests/fail_structure.rs

```diff
@@ -1,9 +0,0 @@
-use node_bindgen::derive::node_bindgen;
-
- 
-#[node_bindgen]
-struct Example4{}
-
-fn main() {
-    
-}
```

Comment:
Parser doesn't support `Enum` and `Union`.  That should be include in fail cases

Reply:
Good idea